examples/atlas/log-download.md

```diff
@@ -0,0 +1,9 @@
+# Download and Unzip Logs
+
+You can download and unzip logs in one go like
+
+```bash
+mongocli atlas logs download <processName> mongodb.gz \
+    --out /dev/stdout \
+    --force | gunzip
```

Comment:
While this usage does work, it is not discoverable and bizarre.

So is the error message produced without --force:

```
serene% ./bin/mongocli  atlas logs download atlas-yesi3n-shard-00-02.l2zw7.mongodb.net mongodb.gz     --out /dev/stdout --projectId 5ed54e14ebe5a05d6fa9a68f
Error: open /dev/stdout: file exists
```

Reply:
Also, according to process list the argument here is hostname, not process name:

```
    [2] {
                 "created" => "2020-06-01T20:12:37Z",
                "hostname" => "atlas-yesi3n-shard-00-02.l2zw7.mongodb.net",
                      "id" => "atlas-yesi3n-shard-00-02.l2zw7.mongodb.net:27017",
               "last_ping" => "2020-10-08T19:54:53Z",
                    "port" => 27017,
              "project_id" => "5ed54e14ebe5a05d6fa9a68f",
        "replica_set_name" => "atlas-yesi3n-shard-0",
               "type_name" => "REPLICA_SECONDARY",
              "user_alias" => "tls-testing-42-shard-00-02.l2zw7.mongodb.net",
                 "version" => "4.2.10"
    }
```

Reply:
> While this usage does work, it is not discoverable and bizarre.

I don't disagree with you but this is what we could offer as a short term solution to allow to output to stdout and also to clobber files.

Right now this command in particular is our most used feature and I'm wary of introducing breaking changes to it but please feel free to create an entry on our [feedback forum](https://feedback.mongodb.com/forums/930808-mongodb-cli) where we can prfioritize any major change better